src/components/inputs/ck-editor/ck-editor.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+    <div>
+      <div style="border: 1px solid #eee" v-if="editorConfig" class="vld-parent">
+          <!-- <overlay-loading :active="isUploadingFile" 
+          :can-cancel="false" background-color="rgb(9 9 9)"
+          :is-full-page="false">
+          <km-spinner size="lg" 
+              :message="t('uploadingFile')"></km-spinner>
+      </overlay-loading>
+   -->
+      <ckeditor 
+          v-model="binding"
+          :editor="editor"
+          :config="editorConfig"
+          :tagName="tagName"
+          :disabled="$attrs.disabled"      
+          @ready="onEditorReady"
+          @focus="onEditorFocus"
+          @blur="onEditorBlur"
+          @input="onEditorInput"
+          @destroy="onEditorDestroy"
+        ></ckeditor>
+      </div>
+      <p style="border: 1px solid #eee; border-top: none">
+        {{ t('attachmentMessage') }}
+        <span class="float-end" id="wordCountSec" style="padding-right: 5px">
+          <strong>{{ t('wordCount') }}: {{ wordCount }}</strong></span>
+      </p>
+      <p style="border: 1px solid #eee; border-top: none" v-if="uploadErrors.length">         
+          <div class="alert alert-danger" role="alert">
+            {{ t('uploadError') }} 
+            <ul><li v-for="error in uploadErrors" :key="error">{{ error.file }}</li></ul>
+          </div>
+        </p>
+    </div>
+  </template>
+
+  <script setup> 
+    import  "@ckeditor/ckeditor5-vue/dist/ckeditor.js";
+    import CKEditor   from '@ckeditor/ckeditor5-vue';
+    import ClassicEditor from '@scbd/ckeditor5-build-inline-full/build/ckeditor.js';    
+    import '@scbd/ckeditor5-build-inline-full/build/ckeditor.css';  
+    import { useI18n} from '../../../services/composables/i18n.js'
+    import { computed, onMounted, ref , defineComponent} from 'vue';
+    import { useLogger } from '../../../services/util/index.js';
+    import KmStorageApi from '../../../services/api/km-storage/KmStorage.js';
+
+    //   import OverlayLoading from 'vue3-loading-overlay';
+    //   import 'vue3-loading-overlay/dist/vue3-loading-overlay.css';
+    //   import KmSpinner from '../KmSpinner.vue';
+    
+    const model    = defineModel({type:String, required:true, default:''});
+
+    const props = defineProps( {     
+        tagName  : { type: String, required: false,default: 'div'},     
+        uploadUrl: { type: String, required: false},
+        config   : { type: Object, required: true},
+        identifier: { type: String, required: true}  
```

Comment:
we will have to reconsider the prop `identifier` as this is specific to absch platform